package.json

```diff
@@ -8,7 +8,7 @@
     "build-prod": "yarn build-prod:quiet --progress",
     "build-photon": "webpack --config res/photon/webpack.config.js",
     "lint": "run-p lint-js lint-css",
-    "lint-js": "eslint *.js bin src --report-unused-disable-directives",
+    "lint-js": "eslint *.js bin src --report-unused-disable-directives --cache --cache-strategy content",
```

Comment:
Note that the cache file will be generated on the CI, but because we don't save it between run it won't be used on subsequent run. We may want to save it between runs in the future, if this starts to be a problem for us.

Also the default cache strategy is "metadata", it saves the date and time information for a file, but this isn't very nice with git, because git doesn't save the time information, so when switching branches we would get some new files checked even if they didn't change. This would also be a problem in CI where a fresh clone is done for each run.